Common/src/Pollster/TaskHandler.cs

```diff
@@ -68,6 +69,7 @@ public sealed class TaskHandler : IAsyncDisposable
   private readonly string                                token_;
   private readonly IWorkerStreamHandler                  workerStreamHandler_;
   private          IAgent?                               agent_;
+  private          TimeSpan?                             delayMessage_;
```

Comment:
this naming makes me think that this is a message an not a `TimeSpan`, i'd inverse to `messageDelay_`

Reply:
It is a verb. It is the action of delaying a message.